### ISSUE TITLE
Disabling "Submit crash logs" causes the app to crash

### ISSUE THREAD

**Issue by luc-bw** [bitwarden/android#4191]
### Bitwarden Beta

- [X] I'm using the new native Bitwarden Beta app and I'm aware that legacy .NET app bugs should be reported in [bitwarden/mobile](https://github.com/bitwarden/mobile)

### Steps To Reproduce

1. Go to Settings
2. Go to About
3. Toggle off "Submit crash logs"
4. Leave Bitwarden
5. Open Bitwarden

### Expected Result

Application opens

### Actual Result

Application crashes before launching

### Additional Context

Clearing the app cache and storage is necessary for the application to launch.

### Build Version

2024.10.1 (19393)

### Environment Details

Google Pixel
Android 15

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Unchanged4352** [comment]
I have this bug too

**Comment by Youngblood50** [comment]
```
type: crash
osVersion: google/husky/husky:15/AP3A.241005.015/2024102400:user/release-keys
flags: dev options enabled
package: com.x8bit.bitwarden:19388
process: com.x8bit.bitwarden
processUptime: 124 + 190 ms
installer: dev.imranr.obtainium

java.lang.RuntimeException: Unable to create application com.x8bit.bitwarden.BitwardenApplication: java.lang.IllegalArgumentException: Cannot uproot tree which is not planted: g7.u@9d8e8ba
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7541)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2423)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8736)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ExecInit.main(ExecInit.java:50)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:369)
Caused by: java.lang.IllegalArgumentException: Cannot uproot tree which is not planted: g7.u@9d8e8ba
	at g7.v.a(Unknown Source:212)
	at g7.v.<init>(Unknown Source:82)
	at V7.a.i(Unknown Source:12)
	at X5.v.get(Unknown Source:4419)
	at ub.a.get(Unknown Source:13)
	at com.x8bit.bitwarden.BitwardenApplication.onCreate(Unknown Source:19)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1391)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7535)
	... 11 more
```

**Comment by SaintPatrck** [comment]
Addressed in #4194 and #4195. Fix will be available in our next release. Thank you for reporting your issue.
